# request
group6_interfaces/AgentData[] agents_data
---
# reply
std_msgs/Bool success